Add Update Guide disclaimer (#3370)
ezsystems/developer-documentation: docs/update_and_migration/from_5.0/update_from_5.0.md

No additional steps needed for [[= product_name =]].

!!! note "Database upgrade scripts"

    The `vendor/ibexa/installer/upgrade/db` contains database update scripts.

>     You do not need to run them, they are part of the Translation management LTS Update [configuration procedure](https://doc.ibexa.co/en/5.0/multisite/translations_management/configure_translations_management/#modify-database-schema).

Reviewer: ```suggestion
    The `vendor/ibexa/installer/upgrade/db` directory contains the `ibexa-5.0.9-to-5.0.10.sql` database update script.

    You don't need to run it.
    The tables added by the script are required only by the [Translation Management LTS update](https://doc.ibexa.co/en/5.0/multisite/translations_management/translations_management/) and adding them is part of the Translation management LTS Update [configuration procedure](https://doc.ibexa.co/en/5.0/multisite/translations_management/configure_translations_management/#modify-database-schema).
```

1) Let's name the specific file that shouldn't be run (it's too generic right now). I know it's two scripts with the same name, here I've simplified it to "script" - but maybe you see how to do this in a better way
2) I've tried clarifying a bit more what uses the tables. Currently the  "they are part of the Translation management LTS Update" part is incorrect - the script themselves are not part of the configuration procedure, they are not used

Author: You are absolutely right - there are multiple upgrade scripts and we need to name the one thet is actually offending.

>     You do not need to run them, they are part of the Translation management LTS Update [configuration procedure](https://doc.ibexa.co/en/5.0/multisite/translations_management/configure_translations_management/#modify-database-schema).

Reviewer 2: ```suggestion
    You don't need to run them, they're part of the Translation management LTS Update [configuration procedure](https://doc.ibexa.co/en/5.0/multisite/translations_management/configure_translations_management/#modify-database-schema).
```
